src/Microsoft.AspNet.Mvc.ModelBinding/ModelBindingResult.cs

```diff
@@ -23,23 +23,60 @@ public ModelBindingResult(object model, string key, bool isModelSet)
         }
 
         /// <summary>
-        /// Gets or sets the model associated with this context.
+        /// Creates a new <see cref="ModelBindingResult"/>.
+        /// </summary>
+        /// <param name="model">The model which was created by the <see cref="IModelBinder"/>.</param>
+        /// <param name="key">The key using which was used to attempt binding the model.</param>
+        /// <param name="isModelSet">
+        /// A value that represents if the model has been set by the <see cref="IModelBinder"/>.
+        /// </param>
+        /// <param name="isEmptyModel">
+        /// If <paramref name="isModelSet"/> is <c>true</c> and this value is <c>true</c>, the <paramref name="model"/>
+        /// was set but contains only default values. If <paramref name="isModelSet"/> is <c>true</c> and this value is
+        /// <c>false</c>, the model contains bound properties. Ignored if <paramref name="isModelSet"/> is
+        /// <c>false</c>.
+        /// </param>
+        public ModelBindingResult(object model, string key, bool isModelSet, bool isEmptyModel)
+            : this(model, key, isModelSet)
```

Comment:
should this be the other way around? Typically a constructor with less parameters calls the one with more.